specs/core/1_shard-data-chains.md

```diff
@@ -281,68 +283,7 @@ def get_shard_block_proposer_index(state: BeaconState,
         i += 1
```

Comment:
> What if all validators in a committee have zero effective balance

I think it's still OK because `effective_balance * MAX_RANDOM_BYTE >= MAX_EFFECTIVE_BALANCE * random_byte` has a greater-than-or-equal sign, and 1 in 32 times `random_byte` is zero.